modyn/trainer_server/internal/trainer/remote_downsamplers/remote_uncertainty_downsampling_strategy.py

```diff
@@ -64,19 +65,26 @@ def inform_samples(
         # keep the mapping index<->sample_id
         self.index_sampleid_map += sample_ids
 
-    def _compute_score(self, forward_output: torch.Tensor) -> np.ndarray:
+    def _compute_score(self, forward_output: torch.Tensor, disable_softmax: bool = False) -> np.ndarray:
```

Comment:
why do we add this option?

Reply:
For the unit test. Because if there is a softmax applied i have no way of easily testing whats going on. If we just take the model output i can calculate this straightforwardly and check correctness.